### ISSUE TITLE
Responses from TV possible? Like channel list etc.?

### ISSUE THREAD

**Issue by miriki** [natalan/samsung-remote#67]
Is it possible to get e.g. a list of channels, favorites or apps from the tv? Right now I only see "send_xxx" to the TV, but none of them delivers responses from the TV, not even a "ok" or /"error", if I don't misunderstand. I'm sure there should be some kind of response mechanism. Like: "send key_game" --> "not available" is displayed on my TV screen. The send routine could be able to catch that methinks.
( model 2013: UE 55 F 6470 SS )

**Comment by jsiegenthaler** [comment]
No. This samsung-remote plugin emulates the remote control, and sends the same keypresses as a remote control. The remote control has no means of checking for TV status.

**Comment by jsiegenthaler** [comment]
@natalan can this issue be closed? It was a question and it has been answered. So I believe it can be closed now.
